src/bp/migrations/v12_1_0-1564421788-database_srv_logs_table_has_date_timestamp.ts

```diff
@@ -6,9 +6,9 @@ const TABLE_NAME = 'srv_logs'
 const COLUMN_NAME = 'timestamp'
 const CURRENT_DATE_FORMAT = 'YYYY-MM-DD"T"HH24:MI:SS.MS"Z"'
 
-// const PG_VARCHAR_TYPE = 'character varying'
+const PG_VARCHAR_TYPE = 'character varying'
 // const SQLITE_VARCHAR_TYPE = 'varchar'
-const PG_TIMESTAMP_TYPE = 'timestamp without time zone'
+// const PG_TIMESTAMP_TYPE = 'timestamp without time zone'
```

Comment:
is there a reason why you keep this commented line ?

Reply:
Well I had to setup 4 different environnent just to get these values. Now that I have them, I kind of want to save them or something. I thought a comment was fine.